tests/e2e/test_s3_ftp_streamer.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+import warnings
+from data.archiver.stream import S3FTPStreamer
+from data.archiver.dataclass import DataArchiverResult, FileResult
+
+class TestArchiver(unittest.TestCase):
```

Comment:
This looks like a hardcoded test, too

Reply:
Deleted. Main test case added to test_archiver.